types/ibm_db/index.d.ts

```diff
@@ -222,6 +222,7 @@ export class Pool implements PoolOptions {
   usedPool: object;
   poolsize: number;
   odbc: ODBC;
+  openSync(connStr: string): Database;
```

Comment:
Should it have an optional `options` parameter?

Reply:
No. The Pool.prototype.openSync has only the one connStr param.

See: 
https://github.com/ibmdb/node-ibm_db/blob/ad38da18e56deee23288ba42a3092134c32b0456/lib/odbc.js#L2113

Reply:
Weird - unless I'm misreading, the doc and sample show options.

Reply:
Oh, the docs are for https://github.com/ibmdb/node-ibm_db/blob/ad38da18e56deee23288ba42a3092134c32b0456/lib/odbc.js#L146.